src/app/containers/FrontPageMain/index.jsx

```diff
@@ -88,12 +94,12 @@ const FrontPageMain = ({ frontPageData }) => {
             margins={itemMargins}
           >
             {groups.map((group, index) => (
-              <FrontPageSection
-                key={group.title}
-                group={group}
-                sectionNumber={index}
-              />
+              <Fragment key={group.title}>
+                {group.type === 'useful-links' && <MostReadContainer />}
```

Comment:
Just to be 100% sure, we can only have 1 useful links group on the page?

Reply:
I just confirmed with @jimjohnsonrollings irl, and he said "yes there should only ever be one useful links or none"